fast_carpenter/summary/binned_dataframe.py

```diff
@@ -241,7 +245,7 @@ def merge(self, rhs):
 
 
 def _make_column_labels(weights):
-    labels = [w + ":" + l for l in weight_labels for w in weights]
+    labels = [weight + ":" + label for label in weight_labels for weight in weights]
```

Comment:
Was this a necessary change? Did it break some style guide?

Reply:
tbh, not really necessary, just changed it to be more clear.
I have a vendetta against lower case `l` as single-letter variable.
And since I changed `l` I thought I might as well change `w`